server/controllers/profile.js

```diff
@@ -0,0 +1,115 @@
+const Profile = require("../models/Profile");
+const asyncHandler = require("express-async-handler");
+
+// @route POST /profiles
+// @desc Given parameters passed in create a profile
+// @access Private
+
+exports.Profiles = asyncHandler(async (req, res, next) => {
```

Comment:
Since we are creating a profile here, maybe consider renaming this controller to make more sense? :)